src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -30,4 +30,9 @@ partial class BoundPointerIndirectionOperator
     {
         protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.Operand);
     }
+
+    partial class BoundUsingLocalDeclarations
+    {
+        protected override ImmutableArray<BoundNode> Children => this.LocalDeclarations.AsImmutable<BoundNode>();
```

Comment:
Use [http://source.roslyn.io/#Microsoft.CodeAnalysis/Collections/StaticCast.cs,97d1e4a8dd39ade0,references](http://source.roslyn.io/#Microsoft.CodeAnalysis/Collections/StaticCast.cs,97d1e4a8dd39ade0,references), which will staticly verify the cast is safe at compile time and won't cause the existing immutable array to be boxed into an ienumerable.